### PR TITLE
Hawkular Metrics 0.15.0 - Release

### PR DESCRIPTION
**Pull request by stefannegrea**
Signed-off-by: Stefan Negrea snegrea@redhat.com

### CODE REVIEW

**Comment by pilhuhn** [comment]
We have all(?) content in AsciiDoc - why now Markdown?

File: src/main/jbake/content/blog/2016/05/02/hawkular-metrics-0.15.0.Final-released.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    - Tag based bucketing for data points is supported for counters and gauges [HWKMETRICS-377](https://issues.jboss.org/browse/HWKMETRICS-377), [HWKMETRICS-373](https://issues.jboss.org/browse/HWKMETRICS-373)
+    - New endpoints for filtering and grouping data by tags:
+      - `GET /hawkular/metrics/gauges/{id}/stats/tags/{tags}`
+      - `GET /hawkular/metrics/counters/{id}/stats/tags/{tags}`
+1. **Tags**
+    - Deleting tags only requires the tag keys and not the values, this simplifies the process for tag deletion. For backwards compatibility, the API will still accept name value pairs but will not take the value into account [HWKMETRICS-385](https://issues.jboss.org/browse/HWKMETRICS-385)
+    - A new endpoint was added to query for the available values contained in a tag [HWKMETRICS-197](https://issues.jboss.org/browse/HWKMETRICS-197)
+      - Endpoint: `GET hawkular/metrics/gauge/tags/{tags}`
+      - Example: `GET hawkular/metrics/gauge/tags/hostname:*01` returns `hostname: ["web01prod", "web01qa", "backend01prod", "backend01qa" ]`
```
Comment (burmanm) [inline src/main/jbake/content/blog/2016/05/02/hawkular-metrics-0.15.0.Final-released.md]: My bug in the original ticket, but this query should be `hostname:*01*` to match those tags (or then the returned values should be changed)

**Comment by stefannegrea** [comment]
@pilhuhn, I switched to markdown because I copy directly the content from the Github release (with minimal changes). I talked with @Jiri-Kremser about the fact that I need to convert markdown to asciidoc just for the blog and he pointed me to JBake documentation that notes both are supported.

**Comment by pilhuhn** [comment]
On 4 May 2016, at 14:51, Stefan Negrea wrote:

> @pilhuhn, I switched to markdown because I copy directly the content 
> from the Github release (with minimal changes). I talked with 
> @Jiri-Kremser about the

Which then begs the question why the GH release is in Markdown and not 
AsciiDoc

**Comment by stefannegrea** [comment]
@pilhuhn, because only markdown is supported in the release editor.

**Pull request merged**